### DISCUSSION TITLE
Custom window positions in Snap Panel?

### DISCUSSION THREAD

**Discussion by gshults** [rxhanson/RectanglePro-Community#495] [Q&A] [answered]
I have created some custom window size and positions that I want to add to the Snap Panel, but I can't see how to do it. Is this possible?

**Comment by rxhanson** [comment, marked as answer]
Looks like I have a bug where custom window size & positions won't show up in that dropdown until you restart the app. I'll slate this bug for the next release.

**Reply by gshults** [reply]
Thanks. The restart works.

Could you also please construct the graphic for the custom window position like you have for all your predefined positions?
